docs/_pages/releases.md

```diff
@@ -14,6 +14,7 @@ sidebar:
 * Added `[Not]BeUpperCased` and `[Not]BeLowerCased` to `StringAssertions` to be able to assert that a string is in upper or lower casing or not - [#1357](https://github.com/fluentassertions/fluentassertions/pull/1357).
 * Added `ObjectAssertions<TSubject, TAssertions>` to ease creation of custom assertion classes - [#1371](https://github.com/fluentassertions/fluentassertions/pull/1371).
 * Added `ComparingBy{Members,Value}(Type)` to allow specifying open generic types - [#1389](https://github.com/fluentassertions/fluentassertions/pull/1389).
+* Added overload of `CollectionAssertions.NotBeEquivalenTo` that takes a `config` parameter` - [#1408](https://github.com/fluentassertions/fluentassertions/pull/1408).
```

Comment:
Typo ;-)

Reply:
It took me several re-reads to spot the missing `t` in `NotBeEquivalenTo`.
I hope that's what you meant?

Reply:
Yes ;-)